src/pages/workspaces/List.js

```diff
@@ -176,7 +176,7 @@ export const WorkspaceList = () => {
       rowCount: sortedWorkspaces.length,
       tableName: currentTab?.tableName || 'workspaces',
       noContentRenderer: () => Utils.cond(
-        [loadingWorkspaces, () => null],
+        [loadingWorkspaces, () => 'Loading...'],
```

Comment:
Nit: This seems like an unrelated change. Is that necessary for this PR? (I am fine with it going in, just want to make sure it is necessary)

Reply:
The table previously displayed nothing while it was loading. But now with the new functionality, the new default value "Nothing to display" will kick in if the renderer function returns something falsey. That's not quite right in this case, so I changed it to something more accurate.